persim/visuals.py

```diff
@@ -280,3 +280,178 @@ def wasserstein_matching(I1, I2, matchidx, palette=None, labels=["dgm1", "dgm2"]
             plt.plot([I1[i, 0], I2[j, 0]], [I1[i, 1], I2[j, 1]], "g")
 
     plot_diagrams([I1, I2], labels=labels, ax=ax)
+
+class Barcode():
+    def __init__(self, diagrams):
+        '''
+        parameters:
```

Comment:
Please follow the doc formatting convention used elsewhere: https://github.com/scikit-tda/persim/blob/989e5c62870743501cd6e9de8df7e03bd1b9f9a6/persim/bottleneck.py#L26

Specifically, `parameters:` to `Parameters` and same for the other headers.